TROUBLESHOOTING.md

```diff
@@ -88,16 +88,20 @@ If Keypass connection is not available the following error entry will appear in
 time=24/Feb/2015 10:49:27 | lvl=ERROR | component=Orchestrator | msg=keypass endpoint not found
 time=24/Feb/2015 10:49:27 | lvl=INFO | component=Orchestrator | msg=Checking endpoints ERROR keypass endpoint not found
 ```
-
+Other optional endpoints as orion, perseo, openldap, mailer or mongodb are also checked, but if a connection error hanpens then a warn log is producec, since these endpoints are optionals.
```

Comment:
Typos: hanpens, producec, optionals.

Reply:
Fixed in 857545c04e5aed69596cd42b3b6d6f2df9ccd67a